src/librustdoc/html/static/main.js

```diff
@@ -2257,9 +2257,9 @@ function defocusSearchBar() {
 
         function implHider(addOrRemove, fullHide) {
             return function(n) {
-                var is_method = hasClass(n, "method") || fullHide;
-                if (is_method || hasClass(n, "type")) {
-                    if (is_method === true) {
+                var should_hide = hasClass(n, "method") || hasClass(n, "associatedconstant");
+                if (should_hide === true || fullHide === true || hasClass(n, "type")) {
+                    if (should_hide === true || fullHide === true) {
```

Comment:
I think keeping javascript naming would be more preferable.

```suggestion
                var shouldHide = hasClass(n, "method") || hasClass(n, "associatedconstant");
                if (shouldHide === true || fullHide === true || hasClass(n, "type")) {
                    if (shouldHide === true || fullHide === true) {
```